modules/sdk-coin-avaxp/test/unit/avaxp.ts

```diff
@@ -510,7 +532,7 @@ describe('Avaxp', function () {
       const txParams = {
         recipients: [
           {
-            address: EXPORT_P_2_C_VERIFY.receiveAddress,
+            address: '',
```

Comment:
this empty string expected?

Reply:
It's expected, `address` is required in `recipients`. 
We could change that in the future